### PR TITLE
Explicit example for adding commands

### PR DESCRIPTION
**Pull request by raphaelstolt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,10 +42,12 @@ $app['console']->run();
 ### Silex
 
 To use the Console Service Provider in a Silex application register the
-Console Service Provider Silex adapter.
+Console Service Provider Silex adapter. To hook custom command-line commands into a Silex application
+add them to Cilex's container aware application instance `$app['console']`.
```
Comment (jmikola) [inline README.md]: It's up to @mvriel, but I think an 80-character line width is preferable.

**Comment by simensen** [comment]
I think I'd rather see this instead of a `foreach` loop:

``` php
<?php
$app['console']->add(new Command\XyzInfoCommand());
$app['console']->add(new Command\XyzSnapshotCommand());
```

Maybe I'm missing something but I don't think the `foreach` loop adds much to this example except extra lines of code. Just adding the new commands directly seems less complicated. I think maybe the `foreach` loop is used in other examples so maybe consistency would be nice but maybe we could update those instances instead of propagating this style?

**Comment by jmikola** [comment]
I concur that the `foreach` isn't necessary. The parallel in Silex's documentation is likely [registering service providers](http://silex.sensiolabs.org/doc/providers.html).

**Comment by raphaelstolt** [comment]
@mvriel feel free to merge or close this pull request.

**Pull request merged**